packages/vats/test/bootstrapTests/bench-vaults-performance.js

```diff
@@ -159,15 +167,47 @@ test.skip('stress vaults', async t => {
   };
 
   // clear out for a baseline
-  await snapshotHeap('start');
+  await collectStats('start', dumpHeap);
   // 10 is enough to compare retention in heaps
   await openN(10, 1);
-  await snapshotHeap('round1');
+  await collectStats('round1', dumpHeap);
   await openN(10, 2);
-  await snapshotHeap('round2');
+  const memStats = await collectStats('round2', dumpHeap);
 
   // let perfObserver get the last measurement
   await eventLoopIteration();
 
+  const name = `stress-vaults${dumpTag}`;
+  let where = import.meta.url;
+  const idx = where.lastIndexOf('agoric-sdk/');
+  if (idx > 0) {
+    where = where.substring(idx);
+  }
+  const benchmarkReport = {
+    ...rows[1],
```

Comment:
I recommend keeping it in milliseconds and letting datadog show the graph and figure out the right time interval to show based on the range of numbers in the data.

Reply:
@toliaqat writes:
> I recommend keeping it in milliseconds and letting datadog show the graph and figure out the right time interval to show based on the range of numbers in the data.

How can datadog know what units to put on the scale? We have to tell it somewhere, maybe when we configure the dashboard? Which would be a unit/type hint that lives outside the tree, and won't help someone running `yarn bench` understand their numbers.

So I disagree, but if you do stick with milliseconds, then at least please change the property name to `msPerVault` or *something* to provide a hint as to the units. Unit-less numbers are horrible, unit-less numbers that use non-primary units are really horrible, unit-*annotated* non-primary-unit numbers are at least a non-ambiguous flavor of horrible.

Reply:
I agree that the units should be labelled, regardless of which units we use.  That seems like table stakes and I will tweak the code accordingly.

In the case of perf measurements I'm inclined towards using milliseconds on ergonomic grounds: as people are gauging the effectiveness of code interventions, the deltas induced by various changes are most likely going to be in the millisecond range rather than in the second range.  It's considerably easier to think about the results you're getting when you can round them to integers in your head and not be too far off.